inst/rmarkdown/templates/glossa/skeleton/glossa.csl

```diff
@@ -0,0 +1,630 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file still used ? 

I don't think so. Can we remove ?

Reply:
Yes! I removed it